keyboards/sthlmkb/litl/info.json

```diff
@@ -0,0 +1,50 @@
+{
+    "manufacturer": "STHLM kb",
+    "keyboard_name": "litl",
+    "url": "https://github.com/mohoyt/litl",
+    "maintainer": "mohoyt",
+    "bootloader": "caterina",
+    "diode_direction": "ROW2COL",
+    "features": {
+         "bootmagic": false,
+         "command": false,
+         "console": false,
+         "extrakey": true,
+         "debug": false,
+         "mousekey": false,
+         "encoder": true,
+         "oled": true
+    },
+    "matrix_pins": {
+         "cols": ["F6", "F7", "B1", "B3", "B2", "B6"],
+         "rows": ["B4", "B5", "D4", "C6", "D7", "E6", "F5", "F4"]
+     },
+    "processor": "atmega32u4",
+    "usb": {
+         "device_version": "1.0.0",
+         "pid": "0x5353",
+         "vid": "0x1729"
```

Comment:
This vendor ID is in use.  Please change.  Anything about 0x4000 is generally okay, but the higher, the better. 

https://www.the-sz.com/products/usbid/index.php?v=0x1729&p=&n=

Reply:
Updated to a unique vendor ID. Thanks!